src/traffic_layout/engine.h

```diff
@@ -29,8 +29,9 @@
 #include <string>
 #include <unordered_map>
 
-#define RUNROOT_WORD_LENGTH 10
-#define COPYSTYLE_WORD_LENGTH 12
+static constexpr unsigned RUNROOT_WORD_LENGTH   = 10;
```

Comment:
Why not declare these as `std::string_view` instances to unify both size and text information? E.g.
```
static const std::string_view RUNROOT_WORD{"--runroot"};
```
The `RUNROOT_WORD_LENGTH` is `RUNROOT_WORD.size()` and comparisons are to `RUNROOT_WORD`.